doc/whats_new/v0.24.rst

```diff
@@ -54,6 +54,16 @@ Changelog
 - |Fix| Better contains the CSS provided by :func:`utils.estimator_html_repr`
   by giving CSS ids to the html representation. :pr:`19417` by `Thomas Fan`_.
 
+:mod:`sklearn.tree`
+.......................
```

Comment:
This should come right after `sklearn.semi_supervised` to be in alphabetical order.

```suggestion
:mod:`sklearn.tree`
...................
```